inc/admin.php

```diff
@@ -149,20 +196,53 @@ function settings_screen() {
  * @param  array $error {
  *     Array of error message components.
  *
- *     @type string $type    Type of error. Typically 'warning' or 'error'.
  *     @type int    $line    Line number of the error.
- *     @type string $message Error message.
+ *     @type string $type    Type of error.
+ *     @type string $value   Optional. Value in question.
  * }
  *
  * @return string       Formatted error message.
  */
 function format_error( $error ) {
+	$messages = get_error_messages();
+
+	if ( ! isset( $messages[ $error['type'] ] ) ) {
+		return __( 'Unknown error.', 'adstxt' );
+	}
+
+	if ( ! isset( $error['value' ] ) ) {
+		$error['value' ] = '';
+	}
+
+	$message = sprintf( esc_html( $messages[ $error['type'] ] ), '<code>' . esc_html( $error['value'] ) . '</code>' );
+
 	/* translators: Error message output. 1: Line number, 2: Error message */
 	$message = sprintf(
 		__( 'Line %1$s: %2$s', 'ads-txt' ),
-		$error['line'],
-		$error['message']
+		esc_html( $error['line'] ),
+		$message // This is escaped piece-wise above and may contain HTML (code tags) at this point
```

Comment:
.... and **will** contain code tags...

Reply:
or 'contains'

Reply:
@adamsilverstein A few of the error messages don't interpolate a value :) I don't know if they'll ever contain anything besides code tags but I figured I better leave it somewhat open-ended.

Reply:
ok!